stdlib/public/runtime/MetadataImpl.h

```diff
@@ -411,6 +411,8 @@ struct BridgeObjectBox :
       
   static void *retain(void *obj) {
     (void)swift_bridgeObjectRetain(obj);
+    // The return value from swift_bridgeObjectRetain may not include the
```

Comment:
Would it make more sense to change swift_bridgeObjectRetain() instead?

Reply:
Yes that makes sense. I did some git spelunking. The function originally (at some point in history) was:

```
void *swift::swift_bridgeObjectRetain(void *object) {
#if SWIFT_OBJC_INTEROP
  if (isObjCTaggedPointer(object))
    return object;
#endif

  auto const objectRef = toPlainObject_unTagged_bridgeObject(object);

#if SWIFT_OBJC_INTEROP
  if (!isNonNative_unTagged_bridgeObject(object))
    return swift_retain(static_cast<HeapObject *>(objectRef));
  return objc_retain(static_cast<id>(objectRef));
#else
  return swift_retain(static_cast<HeapObject *>(objectRef));
#endif
}
```

So my concern when I proposed the fix to BridgeObjectBox that somebody relies on the different return value is probably not justified.